courses/machine_learning/images/mnist_models.ipynb

```diff
@@ -0,0 +1,1012 @@
+{
```

Comment:
<p>Running this cell gave the following error (again I am running the notebook locally in a venv):</p><p> </p><p>Creating temporary tarball archive of 11 file(s) totalling 16.0 KiB before compression.</p><p>Uploading tarball of [mnist_trainer/] to [gs://dherin-sandbox_cloudbuild/source/1571070014.64-98d1f1cd2cf94a1fb95d33048fe41f20.tgz]</p><p>API [cloudbuild.googleapis.com] not enabled on project [146533517870].</p><p>Would you like to enable and retry (this will take a few minutes)?</p><p>(y/N)?</p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/608/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>